lib/exandra/connection.ex

```diff
@@ -785,10 +791,44 @@ defmodule Exandra.Connection do
     |> Enum.map(fn {_, name, _, opts} -> %{name: name, opts: opts} end)
   end
 
-  def column_definitions([]), do: raise(RuntimeError, "you must define at least one column")
+  def alter_column_definitions([]), do: raise(RuntimeError, "you must define at least one column")
+
+  def alter_column_definitions(columns) do
```

Comment:
Prolly fine to make this private?

```suggestion
  defp alter_column_definitions([]), do: raise(RuntimeError, "you must define at least one column")

  defp alter_column_definitions(columns) do
```